src/plugins/videojs-http-source-selector/components/SourceMenuButton.js

```diff
@@ -28,7 +27,9 @@ class SourceMenuButton extends MenuButton {
     }
 
     // Bind update to qualityLevels changes
-    this.player().qualityLevels().on(['addqualitylevel'], videojs.bind(this, this.update));
+    this.player().qualityLevels().on('addqualitylevel', (e) => {
```

Comment:
is e a refernce to an event here or is it something else? i do not recommend e even if it is referencing an event

Reply:
Gladly changed, but honestly `e` is so common for event callbacks (in this repo and in general, like in [this MDN page](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_objects)) that I never thought it could be confusing to anyone.